cleanlab/object_detection/summary.py

```diff
@@ -34,6 +43,412 @@
+
+    class_names: optional
+        Optional dictionary mapping one-hot-encoded class labels back to their original class names in the format ``{"integer-label": "original-class-name"}``.
+
+    class_to_show: optional
+        The number of classes to show in the plots. Classes over `class_to_show` are hidden.
+    """
+    try:
+        import matplotlib.pyplot as plt
+    except ImportError as e:
+        raise ImportError(
+            "This functionality requires matplotlib. Install it via: `pip install matplotlib`"
+        )
+
+    lab_boxes, pred_boxes = bounding_box_size_distribution(
+        labels, predictions, class_names=class_names
+    )
+
+    for i, c in enumerate(lab_boxes.keys()):
+        if i >= class_to_show:
+            break
+        fig, axs = plt.subplots(1, 2, figsize=(10, 5))
+        fig.suptitle(f"Size distributions for bounding box for class {c}")
+        for i, l in enumerate([lab_boxes, pred_boxes]):
+            axs[i].hist(l[c], bins="auto")
+            axs[i].set_xlabel("box area (pixels)")
+            axs[i].set_ylabel("count")
+            axs[i].set_title("annotated" if i == 0 else "predicted")
+
+        plt.show()
+
+
+def plot_class_distribution(labels, predictions, class_names=None):
+    """
+    Plots the distribution of class labels associated with all annotated bounding boxes and predicted bounding boxes in the dataset.
+
+    This plot can help you understand which classes are: rare or over/under-predicted by the model overall.
+
+    Parameters
+    ----------
+    labels:
+        Annotated boxes and class labels in the original dataset, which may contain some errors.
+        Refer to documentation for this argument in :py:func:`object_counts_per_image <cleanlab.object_detection.summary.object_counts_per_image>` for further details.
+
+    predictions:
+        Predictions output by a trained object detection model.
+        Refer to documentation for this argument in :py:func:`object_counts_per_image <cleanlab.object_detection.summary.object_counts_per_image>` for further details.
+
+    class_names: optional
+        Optional dictionary mapping one-hot-encoded class labels back to their original class names in the format ``{"integer-label": "original-class-name"}``.
+    """
+    try:
+        import matplotlib.pyplot as plt
+    except ImportError as e:
+        raise ImportError(
+            "This functionality requires matplotlib. Install it via: `pip install matplotlib`"
+        )
+
+    lab_dist, pred_dist = class_label_distribution(labels, predictions, class_names=class_names)
+    fig, axs = plt.subplots(1, 2, figsize=(10, 5))
+    fig.suptitle(f"Distribution of classes in the dataset")
+    for i, d in enumerate([lab_dist, pred_dist]):
+        axs[i].pie(d.values(), labels=d.keys(), autopct="%1.1f%%")
+        axs[i].set_title("Annotated" if i == 0 else "Predicted")
+
+    plt.show()
+
+
+def plot_class_accuracy(labels, predictions, class_names=None):
```

Comment:
Can we make class_names take in a limited dictionary and only plot those classes? For example if data has classes 0,1,2 which are car,person,cup respectively if you pass in class_names={0: car} then plot_class_accuracy will only plot the car class. This way if there are many classes a user can selectively plot the top classes instead. 

Remember to change docstring definition :D